pkg/operator/ceph/object/spec.go

```diff
@@ -155,6 +155,21 @@ func (c *clusterConfig) makeRGWPodSpec(rgwConfig *rgwConfig) (v1.PodTemplateSpec
 		ServiceAccountName: serviceAccountName,
 	}
 
+	// If the rgw ops logs are enabled we add the side-car container
+	if c.store.Spec.RGWOpsLogEnabled {
+
+		monStore := cephconfig.GetMonStore(c.context, c.clusterInfo)
+		if err := monStore.Set("client.rgw", "rgw_enable_ops_log", "true"); err != nil {
+			logger.Error(err, "failed to apply logging configuration for rgw ops log")
+		}
+
+		logger.Info("rgw ops log is enabled now, creating side-car for it...")
```

Comment:
`logger` lines should always include the namespaced name of the resource so that users with multiple clusters/stores can identify logs specific for each resource.

Aside from this, I don't think this log line is critical. I think debug level is appropriate, or disabling it could be fine also.